src/cgame/rocket/rocket_documents.cpp

```diff
@@ -56,6 +57,15 @@ void Rocket_LoadDocument( const char *path )
 
 }
 
+// Scale the UI proportional to the screen size
+void Rocket_SetDocumentScale( Rml::Core::ElementDocument& document )
+{
+	// This gets 12pt on 1920�1080 screen, which is libRocket default for 1em
```

Comment:
Nitpick: not sure it's worth using the UTF8 character for `×` here, looks like just asking for problems to me (even if should not do problems to most).

Reply:
true